### ISSUE TITLE
installationId not sent with ParseUser.login()

### ISSUE THREAD

**Issue by Danaru87** [parse-community/Parse-SDK-Flutter#373]
Hello !

When ParseUser is created and loggedIn with .signUp(), the new _Session object is created with installationId set.

But, using .login(), a new session is created without installationId.

![parse session](https://user-images.githubusercontent.com/5873203/82051577-7046e800-96ba-11ea-85f9-aa7905a6171d.png)

**Version**
> Flutter : 1.17.0
> 
> Parse Server: 4.2.0
> 
> parse_server_sdk: ^1.0.26

**Comment by Danaru87** [comment]
**In parse_user.dart:**
Login does not retrieve installationId and send it to parse server.

```dart
Future<ParseResponse> login() async {
    try {
      final Map<String, dynamic> queryParams = <String, String>{
        keyVarUsername: username,
        keyVarPassword: password
      };

      final Uri url = getSanitisedUri(_client, '$keyEndPointLogin',
          queryParams: queryParams);
      _saveChanges();
      final Response response =
          await _client.get(url, headers: <String, String>{
        keyHeaderRevocableSession: '1',
      });

      return await _handleResponse(
          this, response, ParseApiRQ.login, _debug, parseClassName);
    } on Exception catch (e) {
      return handleException(e, ParseApiRQ.login, _debug, parseClassName);
    }
```

Looking more closely, it seems that the _applicationId property is never saved into ParseCoreData after new ParseInstallation creation.

**Comment by Danaru87** [comment]
Fixed in release/1.0.27
